ui/src/app/cohort-search/overview/overview.component.tsx

```diff
@@ -133,6 +133,9 @@ const styles = reactStyles({
   },
 });
 
+// Limit the size of cohort definition to 1MB
+const COHORT_BYTE_LIMIT = 1000000;
```

Comment:
Change this value to `1000` or less for testing. Then a few items should trigger the warning.